src/NexyCrypt.php

```diff
@@ -299,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = [])
+    private function request($method, $uri, array $options = ['verify' => false])
```

Comment:
Yeah, but passing an array has default value is not a very good choice.

We have to find something else.

Maybe using the option resolver? http://symfony.com/doc/current/components/options_resolver.html

But I'm not sure. I think we have to define all possible options in this case.

Or just a property with a default value which can be changed if we have this option?